linux/manscript.sh

```diff
@@ -0,0 +1,23 @@
+# !/bin/bash
+echo "Executing script"
+
+# Check if help2man installed
+if [[ $(dpkg -l | grep help2man) ]]; then
+    echo "help2man found."
+else
+    echo "help2man does not exist."
+    echo "Please install help2man. (for Ubuntu use: sudo apt-get install help2man)"
+    exit 1
+fi
+
+exechelp2man="help2man --section 7 --include=replace.h2m ./ccextractor > ccextractor.7" 
+echo $exechelp2man
+eval $exechelp2man
+
+mkdir -p /usr/local/man/man7
+sudo cp ccextractor.7 /usr/local/man/man7/ 
```

Comment:
Assuming a user has sudo available doesn't seem to wise. It'd be better to request the script to be executed with sudo or as root instead, and have a check for it.